csfieldguide/templates/general/index.html

```diff
@@ -10,10 +10,10 @@
   <div class="jumbotron jumbotron-fluid py-5">
     <div class="container text-center">
       <div class="row align-items-center">
-        <div class="col-6 mx-auto col-md-4 order-md-2 csfg-logo-center">
-          <img id="homepage-logo" src="{% static 'img/logos/guide-logo-light.png' %}">
+        <div class="col-6 mx-auto col-md-4 order-md-2">
+          <img class="csfg-logo-image" id="homepage-logo" src="{% static 'img/logos/guide-logo-light.png' %}">
```

Comment:
This seems to be an odd way to select the image for CSS. Why use a class (which is used for multiple elements), when it already has an ID (an unique identifier)?

Reply:
It was a class originally so I stuck with that. Will change

Reply:
Misclicked mid-commit, but have added the attributes to _homepage.scss